src/mmw/js/src/compare/templates/compareModificationsPopover.html

```diff
@@ -20,7 +20,7 @@ <h5>{{ caption }}</h5>
 
 {% macro gwlfeModificationsTable(caption, mods) %}
 <table class="compare-modifications-table -gwlfe">
-{% if modifications.length > 0 %}
```

Comment:
Curious, was this just an existing undiagnosed but?

Reply:
Yeah 😬